contracts/interfaces/IInterchainGasEstimation.sol

```diff
@@ -23,10 +23,11 @@ interface IInterchainGasEstimation {
     }
 
     struct GasInfo {
-        GasEstimationType gasEstimationType; // Custom gas pricing rule, such as L1 data fee on L2s
```

Comment:
i prefer using the type here for cleaner code. the remaining fields can still get packed